### PR TITLE
Russ/marlowe finance redesign

### PR DESCRIPTION
**Pull request by RussellHepton**
These are the first defining changes to the Marlowe home page website redesign.

Pre-submit checklist:
- Branch
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: marlowe-website/src/index.njk [hunk 1]
```diff
@@ -15,245 +15,414 @@
...
-          <p class="text-lg ">Tell us what you think, we need your feedback to turn great into exceptional.</p>
-          <div class="flex items-end justify-end">
-            <img class="relative top-2" src="{% static './static/img/top-down-arrow.svg' %}" alt="Arrow">
-            <button class="text-sm flex-shrink-0 primary-button">Submit feedback</button>
+        <!--Row-->
+
+      </div>
+
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:135]: unneeded comment

File: marlowe-website/src/index.njk [hunk 2]
```diff
@@ -15,245 +15,414 @@
...
+
+        <div class="relative w-full max-w-screen-widest">
+          {#Defines the width#}
+          <!--Row-->
+          <div class="">
+            <!--Column-->
+
+            <h3 class="text-lg font-semibold mt-5 pb-4 md:block hidden">Try the first of many smart contract templates, more coming soon.</h3>
+            <img loading="lazy" class="w-2/3 mx-auto lg:pb-0 md:pb-8 pb-8 hidden md:block" src="{% static './static/img/run-tabs.png' %}" alt="">
+            <img loading="lazy" class="w-2/3 mx-auto lg:pb-0 md:pb-8 pb-8 block md:hidden" src="{% static './static/img/tabs-mobile.jpg' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:145]: alt text

File: marlowe-website/src/index.njk [hunk 3]
```diff
@@ -15,245 +15,414 @@
...
+            <!--Column-->
+
+            <h3 class="pt-16 text-xl font-semibold pb-4 md:block hidden">See how it works</h3>
+
+            <div class="relative xl:w-3/4 w-full mx-auto">
+              <!--Video thumbnail-->
+              <div class="absolute md:bottom-5 md:right-5 bottom-3 right-3">
+                <a href="#" class="scale"><img loading="lazy" class="w-full" src="{% static './static/img/marlowe-play-btn.svg' %}" alt=""></a>
               </div>
+              <img loading="lazy" class="w-full mx-auto" src="{% static './static/img/run-video-thumbnail.png' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:169]: alt text

File: marlowe-website/src/index.njk [hunk 4]
```diff
@@ -15,245 +15,414 @@
...
-            </div>
-          </div>
+          <!--Column-->
         </div>
+        <!--Row-->
+
       </div>
+
+      <img loading="lazy" class="section-circle-top-right invisible lg:visible" src="{% static './static/img/circle-top-right.svg' %}" alt="xx">
+      <img loading="lazy" class="section-circle-bottom-left invisible lg:visible" src="{% static './static/img/circle-bottom-left.svg' %}" alt="xx">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:199]: alt text

File: marlowe-website/src/index.njk [hunk 5]
```diff
@@ -15,245 +15,414 @@
...
-      {# For displaying this banner we use a background image in the second element if its small or a
-      2 element grid if it's big. I tried to put them both as grid or as background image but I couldn't
-      make it work that way. #}
-      <div class="grid grid-cols-1 md:grid-cols-2 widest:mx-auto max-w-screen h-full">
-        <div class="bg-market-banner md:bg-none bg-no-repeat bg-cover bg-pos-market-banner-mobile flex flex-col i items-start pl-4 md:pl-0 md:items-center pt-8 md:pt-0 md:justify-center">
-          <img loading="lazy" class="h-play-img-sm md:h-play-img-md" src="{% static './static/img/marlowe-market-coming-soon.svg' %}" alt="Marlowe play Update coming 2021 on the third or fourth quarter">
-        </div>
-        <div class="hidden md:block w-max">
-          <img loading="lazy" src="{% static './static/img/marlowe-market-banner-background-desktop.png' %}" alt="Woman using marlowe market on her cellphone">
+    <!--S1 - Marlowe Run-->
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:203]: the comments are a little misleading, is this marlowe run or build section?

File: marlowe-website/src/index.njk [hunk 6]
```diff
@@ -15,245 +15,414 @@
...
-          <h4 class="font-bold">Designed for finance</h4>
-          <p>Marlowe contracts are written in the language of finance.</p>
 
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative grid grid-cols-1 lg:grid-cols-2 w-full max-w-screen-widest z-10">
+          <!--Row-->
+
+          <div class="lg:w-10/12 w-11/12 self-center justify-self-center">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:231]: are you using `w-10/12` and `11/12` for margins? if so please change to use proper margins.

File: marlowe-website/src/index.njk [hunk 7]
```diff
@@ -15,245 +15,414 @@
...
+            <!--Left column-->
+            <h3 class=" text-green lg:text-xl text-lg font-semibold lg:mt-0 md:mt-5 pb-4">No code? No problem.</h3>
+            <h4 class="font-comfortaa-bold md:text-3xl text-27px mb-4">Write smart contracts in the visual builder</h4>
+            <p class="md:text-xl text-lg md:mb-0 mb-12">With the Blockly visual builder you can develop financial smart contracts whithout writing a single line of code, right in the browser.</p>
+            <div class="relative md:w-5/6 w-11/12 md:mt-14 mt-8">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:236]: same comment about margins and width here

File: marlowe-website/src/index.njk [hunk 8]
```diff
@@ -15,245 +15,414 @@
...
+              <!--Row-->
+              <div class="relative w-full">
+                <!--Column-->
+                <a href="https://staging.marlowe-dash.iohkdev.io" target="_blank" class="primary-button text-lg">Try it</a>
+                <img loading="lazy" class="w-32 absolute left-52 bottom-2 xl:block hidden" src="{% static './static/img/arrow-light.svg' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:241]: alt text

File: marlowe-website/src/index.njk [hunk 9]
```diff
@@ -15,245 +15,414 @@
...
+              </div>
+              <!--Column-->
+            </div>
+          </div>
+          <!--Left column-->
+
+          <div class="lg:row-start-auto row-start-1 w-11/12 mx-auto mt-4 mb-8 md:py-16 py-10 px-5 bg-white shadow-lg h-auto rounded-lg">
+            <img loading="lazy" class="lg:w-11/12 md:w-3/4 w-11/12 lg:h-80 md:h-80 h-60 object-cover mx-auto" src="{% static './static/img/blockly.gif' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:249]: same width and margin comment
alt text

File: marlowe-website/src/index.njk [hunk 10]
```diff
@@ -15,245 +15,414 @@
...
+          </div>
+
+          <!--<div class="lg:row-start-auto row-start-1"> <img loading="lazy" class="w-full" src="{% static './static/img/marlowe-blockly-blocks.png' %}" alt=""> </div>-->
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:252]: alt text

File: marlowe-website/src/index.njk [hunk 11]
```diff
@@ -15,245 +15,414 @@
...
+
+          <div class="">
+            <!--Column-->
+            <h3 class="pt-16 lg:text-xl text-lg font-semibold pb-4 md:block text-green">See how it works</h3>
+            <div class="relative xl:w-3/4 w-full mx-auto">
+              <!--Video thumbnail-->
+              <div class="absolute md:bottom-5 md:right-5 bottom-3 right-3">
+                <a href="#" class="scale"><img loading="lazy" class="w-full" src="{% static './static/img/marlowe-play-btn.svg' %}" alt=""></a>
+              </div>
+              <img loading="lazy" class="w-full mx-auto" src="{% static './static/img/marlowe-build-video-thumbnail.png' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:272]: alt text

File: marlowe-website/src/index.njk [hunk 12]
```diff
@@ -15,245 +15,414 @@
...
-            <div class="bg-mac-yellow w-3 h-3 rounded-full mr-2"></div>
-            <div class="bg-mac-green w-3 h-3 rounded-full"></div>
+
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative grid grid-cols-1 lg:grid-cols-2 w-full max-w-screen-widest  mt-8 z-10">
+          <!--Row-->
+
+          <div class="w-11/12 mx-auto my-10 shadow-lg h-auto rounded-lg" style="background-color:#232d3d;">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:290]: if that colour is not already present, you should add it to our colour pallete

https://tailwindcss.com/docs/customizing-colors

and then use `bg-custom-colour`

Comment (hrajchert) [reply marlowe-website/src/index.njk:290]: also `w-11/12` for margins and `mx-auto` for centering should be changed for `mx-{val}`for margin and `flex justify-center` for centering.

File: marlowe-website/src/index.njk [hunk 13]
```diff
@@ -15,245 +15,414 @@
...
+
+            <div class="bg-darkblue w-full h-20 py-5 md:px-8 px-6 rounded-t-lg relative">
+              <img loading="lazy" class="w-24 float-left" src="{% static './static/img/marlowe-build-dark.svg' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:293]: float-left sounds fishy

Comment (hrajchert) [reply marlowe-website/src/index.njk:293]: also alt text

File: marlowe-website/src/index.njk [hunk 14]
```diff
@@ -15,245 +15,414 @@
...
+              <h3 class="text-green text-right text-lg font-bold pt-2">Marlowe</h3>
+            </div>
+
+            <img loading="lazy" class="md:w-96 w-72 pl-5 py-5 object-cover" src="{% static './static/img/marlowe-code.gif' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:297]: alt text

File: marlowe-website/src/index.njk [hunk 15]
```diff
@@ -15,245 +15,414 @@
...
+
           </div>
-          <div class="rounded overflow-hidden">
-            <img loading="lazy" class="w-full" src="{% static './static/img/marlowe-code.svg' %}" alt="Marlowe contract example">
+          <!--Tailwind class "Rounded" does nothing-->
+
+          <!--<div class="" style=""> <img loading="lazy" class="" src="{% static './static/img/marlowe-build-code-module.png' %}" alt=""> </div>-->
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:302]: remove comments

File: marlowe-website/src/index.njk [hunk 16]
```diff
@@ -15,245 +15,414 @@
...
+
+          <div class="lg:w-10/12 w-11/12 mb-0 md:mt-0 mt-0 md:mb-0 lg:-mx-0 mx-auto lg:justify-self-center self-center" style="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:304]: width and margin issue

File: marlowe-website/src/index.njk [hunk 17]
```diff
@@ -15,245 +15,414 @@
...
+            <!--Right column-->
+            <h3 class=" text-green lg:text-xl text-lg font-semibold pb-4">Write in Marlowe, JS or Haskell</h3>
+            <h4 class="font-comfortaa-bold md:text-3xl text-27px mb-4">Build your very own smart contract ideas</h4>
+            <p class="md:text-xl text-lg lg:mb-14 mb-12">Rapidly build your own smart contracts from scratch or from pre-built templates in the Marlowe Build sandbox.</p>
+            <div class="relative md:w-5/6 w-11/12">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:309]: width instead of margin

File: marlowe-website/src/index.njk [hunk 18]
```diff
@@ -15,245 +15,414 @@
...
+              <!--Row-->
+              <div class="relative w-full">
+                <!--Column-->
+                <a href="https://staging.marlowe-dash.iohkdev.io" target="_blank" class="primary-button text-lgl">Try it</a>
+                <img loading="lazy" class="w-32 absolute left-52 bottom-2 xl:block hidden" src="{% static './static/img/arrow-light.svg' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:314]: alt text

File: marlowe-website/src/index.njk [hunk 19]
```diff
@@ -15,245 +15,414 @@
...
+            <!--Row-->
           </div>
+          <!--Right column-->
+
         </div>
+        <!--Row-->
+
       </div>
 
+      <img loading="lazy" style="" class="section-m-top-right absolute invisible lg:visible w-full" src="{% static './static/img/marlowe-background-m.svg' %}" alt="xx">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:327]: invisible still occupies space, use hidden instead

File: marlowe-website/src/index.njk [hunk 20]
```diff
@@ -15,245 +15,414 @@
...
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative w-full max-w-screen-widest">
+          <!--Row-->
+          <div class="">
+            <!--Column-->
+            <h2 class="md:text-3xl lg:text-4xl flex flex-col md:font-comfortaa-bolder font-comfortaa-bold leading-tight lg:pt-8 pb-4 text-27px">Marlowe Pioneers</h2>
+            <h2 class="text-xl lg:text-2xl xl:text-27px pb-0">Be the first to test and feedback on this groundbreaking new DeFi technology.</h2>
+            <div class="relative mx-auto md:w-5/6 w-11/12 md:mt-18 md:mb-5 mt-12">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:382]: width margin and centering issue

File: marlowe-website/src/index.njk [hunk 21]
```diff
@@ -15,245 +15,414 @@
...
+              <!--Row-->
+              <div class="w-full mx-auto">
+                <!--Column-->
+                <a href="https://staging.marlowe-dash.iohkdev.io" target="_blank" class="relative primary-button text-lg">
+                  <img loading="lazy" class="w-32 absolute -right-36 bottom-6 xl:block hidden" src="{% static './static/img/arrow-light.svg' %}" alt="">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:387]: alt text

File: marlowe-website/src/index.njk [hunk 22]
```diff
@@ -15,245 +15,414 @@
...
+                  Register now
+                </a>
+              </div>
+              <!--Column-->
+            </div>
+            <!--Row-->
+          </div>
+          <!--Column-->
+        </div>
+        <!--Row-->
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:397]: remove comments

File: marlowe-website/src/index.njk [hunk 23]
```diff
@@ -15,245 +15,414 @@
...
+
       </div>
+      <img loading="lazy" style="" class="pink-m-left invisible lg:visible" src="{% static './static/img/pink-m-left.svg' %}" alt="xx">
+      <img loading="lazy" style="" class="pink-m-right invisible lg:visible" src="{% static './static/img/pink-m-right.svg' %}" alt="xx">
```
Comment (hrajchert) [inline marlowe-website/src/index.njk:401]: invisible still occupies space, also alt text

**Review by hrajchert — CHANGES_REQUESTED** [review]

File: marlowe-website/src/index.css [hunk 24]
```diff
@@ -6,13 +6,153 @@
...
+  content: url("/static/img/chevron-external-light.svg");
+  width:24px;
+  background-size: 24px;
+  position: absolute;
+  bottom:6px;
+}
+
+.logo-height {
+  height: 82px;
+}
```
[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css]: height: {
        logo: "100px",
        "logo-wider": "85px",
      },

File: marlowe-website/src/index.css [hunk 25]
```diff
@@ -10,9 +10,133 @@ body {
...
+  background-size: 24px;
+}
+
+.menu-item a {
+  border-bottom: 4px solid white;
+  padding-bottom: 32px;
+}
+
+.menu-item a:hover {
+  border-bottom: 4px solid #273245;
```
[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css:36]: :( Returns "Failed to compile" in browser refresh

File: marlowe-website/src/index.css [hunk 26]
```diff
@@ -10,9 +10,133 @@ body {
...
+}
+
+.section-circle-top-right {
+  position: absolute;
+  display: inline-block;
+  width: 300px;
```
[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css:42]: Changed to: 

.section-circle-top-right {
  position: absolute;
  @apply w-80;
  right: 0;
  top: 0;
  z-index: 0;
}

File: marlowe-website/src/index.css [hunk 27]
```diff
@@ -10,9 +10,133 @@ body {
...
+
+.section-m-top-right {
+  position: absolute;
+  display: inline-block;
+  width: 1440px;
+  right: 0;
+  top: 0;
+  z-index: 0 !important;
+  opacity: 0.8;
+}
```
[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css:65]: Changed the config file to include a new width property
      width: {
        pinkM: "280px",
      },

File: marlowe-website/src/index.css [hunk 28]
```diff
@@ -10,9 +10,133 @@ body {
...
+
+.pink-m-right {
+  position: absolute;
+  display: inline-block;
+  width: 280px;
+  right: 0;
+  bottom: 0;
+  z-index: 0 !important;
+  opacity: 0.8;
+}
```
[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css:85]: .pink-m-left,
.pink-m-right {
  position: absolute;
  left: 0;
  top: 0;
  z-index: 0;
  opacity: 0.8;
}

[the remark this answers is not in the document]
Comment (RussellHepton) [reply marlowe-website/src/index.css:85]: Not quite the same, but maybe this is better? 
.pink-m-left,
.pink-m-right {
  position: absolute;
  z-index: 0;
  opacity: 0.8;
}

.pink-m-left {
  left: 0;
  top: 0;
}

.pink-m-right {
  right: 0;
  bottom: 0;
}

**Pull request merged**